### ISSUE TITLE
Cannot open bunny.stl

### ISSUE THREAD

**Issue by harrychy8** [dilevin/computer-graphics-ray-tracing#4]
![capture](https://user-images.githubusercontent.com/15678482/51771458-0b2b4580-20b7-11e9-8f54-39c31b3110e4.PNG)

**Comment by dilevin** [comment]
1. Could you load bunny.json for assignment 2 or is this specific to assignment 3.
2. Could you try copying bunny.stl to the same directory as the raytracing.exe

**Comment by harrychy8** [comment]
assignment 2 was fine.
I tried copying it to other places, but didn't work either

**Comment by dilevin** [comment]
1.Please define "other places".  

2. Last I checked, the  json file is the same for A2 and A3 aside from the addition of materials. Are you building them in the same way ?

**Comment by wbh123456** [comment]
I have exactly the same problem here.
I tried re-cloned the whole project, using the bunny from assignment 2, put bunny.stl to the same directory as .exe , they all did not work out.

Also, I could not open two-spheres-and-plane and inside-a-sphere. They gave Symbol file not loaded exception.

![image](https://user-images.githubusercontent.com/35285095/51783049-8ade0200-2101-11e9-8c01-a1ea31657eab.png)

**Comment by dilevin** [comment]
Since this seems to be a windows specific problem, for now use a Ubuntu Linux machine to work on the assignment (such as the CDF machines). We will let you know if we manage to debug this.

**Comment by dilevin** [comment]
Ignore the previous fix. The problem is that you are using the wrong path separator.

On windows the the path should be specified with a back slash like this:
..\data\bunny.json

here's an image from the code running on windows 10 in VS2017 (I just made a small version to save time)
<img width="103" alt="image" src="https://user-images.githubusercontent.com/1903521/51789931-97437880-215c-11e9-83c8-5dc266bb4625.png">

Edit: I pushed a patch to the branch 'fix' which might address the second post. To test this run:
git fetch origin
git checkout fix

This should change the main.cpp to use an OS dependent PATH_SEPERATOR. If it works I will push it to the main branch.
